src/release/publisher.ts

```diff
@@ -1190,6 +1246,28 @@ export function isAwsCodeArtifactRegistry(registryUrl: string | undefined) {
   return registryUrl && AWS_CODEARTIFACT_REGISTRY_REGEX.test(registryUrl);
 }
 
+/**
+ * Info extracted from AWS CodeArtifact URL
+ */
+export interface AwsCodeArtifactInfo {
+  readonly domain?: string;
+  readonly account?: string;
+  readonly region?: string;
+}
+
+/**
+ * Parses info about code artifact domain from given AWS code artifact url
+ * @param url Of code artifact domain
+ * @returns domain, account, and region of code artifact domain
+ */
+export function awsCodeArtifactInfoFromUrl(url?: string): AwsCodeArtifactInfo {
```

Comment:
No need for this to be exported.
```suggestion
function awsCodeArtifactInfoFromUrl(url?: string): AwsCodeArtifactInfo {
```